openhands/runtime/utils/runtime_templates/Dockerfile.j2

```diff
@@ -4,10 +4,13 @@ FROM {{ base_image }}
 ENV POETRY_VIRTUALENVS_PATH=/openhands/poetry
 ENV MAMBA_ROOT_PREFIX=/openhands/micromamba
 
-{% if not skip_init %}
+{% if build_from_scratch %}
 # ================================================================
 # START: Build Runtime Image from Scratch
 # ================================================================
+# This is used in cases where the base image is the current OpenHands release, rather
+# than something more generic like nikolaik/python-nodejs
```

Comment:
I'm a bit confused about this comment - I think "build from scratch" is used when the base image is generic image like python-nodejs?

Reply:
For sure - I updated the comment